README.md

```diff
@@ -909,7 +920,6 @@ https://discord.com/channels/469524606043160576/905950538816978974/1134950311719
   * 2024+ Mirai (Speculated from TechInfo lookup)
 * Q: I’ll tell my friends, can we spend marketing budget on new Toyota/lexus ecu cracking? :kekw: geohot: no, buy votes
    * https://discord.com/channels/469524606043160576/954493346250887168/1197701819686715462
-[^1]: As a shameless plug, do you like those real-time updating embedded values from the Google Spreadsheet up there for the bounty and vote tracker? I made [cellshield.info](https://cellshield.info) for that and other non-security key related uses. Check it out and let me know outside of this discussion if you have any comments!
```

Comment:
As far as I can tell, this doesn't show anywhere on the page today. I put it at the bottom and linked it to all the shields.

Reply:
Sounds good to me.